python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -339,6 +366,47 @@ async def setup_laser(
         await self.change_laser_optical_configuration(optical_configuration)
         await self.change_laser_wavelength(wavelength, use_projector)
 
+    async def setup_cbp(
+        self,
+        azimuth: float,
+        elevation: float,
+        mask: typing.Optional[int] = None,
```

Comment:
Do we want this to be optional or have a default here? I think every time we move the CBP it would be good to confirm the mask, focus and rotation.